jenkins/hourly_maintenance.py

```diff
@@ -137,13 +140,41 @@ def KillLoopingBash():
             err |= subprocess.call(['sudo', 'kill', '-9', pid])
     return err
 
+
+def DeleteCorruptGitRepos():
```

Comment:
yeah, certainly not blocking over this. just a suggestion of something that might be nice.